### DISCUSSION TITLE
How to run sidecar container with KubernetesPodOperator

### DISCUSSION THREAD

**Discussion by sumith-tempus** [apache/airflow#26929] [Q&A]
I wanted to run the MS SQL server image along with another GCR image using KubernetesPodOperator. Currently, KubernetesPodOperator is taking the default image passed. How can I run spinup two containers (GCR) inside KubernetesPodOperator?

**Comment by potiuk** [comment]
Look for pod_template_file https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html#difference-between-kubernetespodoperator-and-kubernetes-object-spec
